gorp.go

```diff
@@ -114,6 +114,14 @@ type SqlExecutor interface {
 	queryRow(query string, args ...interface{}) *sql.Row
 }
 
+// DynamicTable allows the users of gorp to dynamically
+// use different database table names during runtime
+// while sharing the same golang struct for in-memory data
+type DynamicTable interface {
+	GetTableName() string
```

Comment:
In keeping with [the getters section of Effective Go](https://golang.org/doc/effective_go.html#Getters), can we rename this to `TableName()`?